test/e2e-go/cli/goal/expect/algokeyExpectCommon.exp

```diff
@@ -0,0 +1,34 @@
+# Algorand Algokey Expect Utility Package
```

Comment:
This file should probably live elsewhere, but I think it's ok to be here for now.

Reply:
yes, probably `test/e2e-go/cli/algokey/expect/expectCommon.exp` along with a series of expect test for the algokey utility.

Reply:
I'll create a ticket for it.